exercise/ackerman_function.py

```diff
@@ -0,0 +1,35 @@
+"""
+This module contains a code for exercises 11-3 related to:
+Think Python, 2nd Edition
+Chapter 11: Dictionaries
+
+Memoize the Ackermann function from Exercise 6-2 and see if memoization makes it
+possible to evaluate the function with bigger arguments.
+"""
+
+dict_variable = {}
+
+def ackermann(m, n):
```

Comment:
Should update the parameter name.
 `i, j, k, m, n, ....`  should only be used in the loop, or temporary counter variables.
 
 Please double-check and apply for other files.